core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -590,7 +590,7 @@ wasm_interp_get_frame_ref(WASMInterpFrame *frame)
 #endif
 
 #if WASM_ENABLE_MEMORY64 != 0
-#define POP_MEM_OFFSET() (is_memory64 ? POP_I64() : POP_I32())
+#define POP_MEM_OFFSET() (is_memory64 ? POP_I64() : (uint32)POP_I32())
```

Comment:
Maybe it's a good idea to PR directly to the main branch, and in main `POP_TBL_ELEM_IDX()` also has this problem.

Reply:
Agree, branch `dev/shared_heap` had been merged to main. And main has resolved the CI wamr-ide test failure.

Reply:
Done